From 35d6f857441535b43e7cfb883b8f580781e40b9e Mon Sep 17 00:00:00 2001
From: Peter Den Hartog <pdenhartog@tormach.com>
Date: Fri, 25 Oct 2024 17:21:56 -0600
Subject: [PATCH] configure.ac: Fix SITEPY check to work on non-debian systems

---
 src/configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/configure.ac b/src/configure.ac
index 91e1bcf297..3a997102a1 100644
--- a/src/configure.ac
+++ b/src/configure.ac
@@ -1684,7 +1684,7 @@ fi
 
 
 AC_MSG_CHECKING(for site-package location)
-SITEPY="$($PYTHON -c 'import sysconfig; s = sysconfig.get_scheme_names(); m=list(set(("deb_system", "rpm_prefix")) & set(s)); print(sysconfig.get_path("platlib", m.__getitem__(0))) if m else print("/usr/lib/python3/dist-packages");')"
+SITEPY="$($PYTHON -c 'import sysconfig; print(sysconfig.get_paths()["purelib"])')"
 AC_MSG_RESULT($SITEPY)
 
 AC_MSG_CHECKING(for working GLU quadrics)
