From 12e22784ec8c5e8be5eb0d1bf325cc4334bf77d6 Mon Sep 17 00:00:00 2001
From: Peter Den Hartog <pdenhartog@tormach.com>
Date: Mon, 28 Oct 2024 11:31:59 -0700
Subject: [PATCH] Makefile: Patch to use /usr/bin/env for Python and TCLSh
 shebangs

---
 src/Makefile | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 965c93a2f5..dda331a966 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -482,7 +482,7 @@ TARGETS += $(patsubst po/%.po, objects/%.msg, $(wildcard po/*.po))
 TARGETS += ../bin/tooledit
 ../bin/tooledit: Makefile
 	$(Q)rm -f $@
-	$(Q)(echo "#!$(TCLSH)")>|$@
+	$(Q)(echo '#!/usr/bin/env tclsh')>|$@
 	$(Q)(echo "source $(EMC2_TCL_LIB_DIR)/tooledit.tcl")>>$@
 	$(Q)(echo "standalone_tooledit")>>$@
 	$(Q)chmod +x $@
@@ -491,7 +491,7 @@ TARGETS += ../bin/tooledit
 TARGETS += ../bin/ngcgui
 ../bin/ngcgui: Makefile
 	$(Q)rm -f $@
-	$(Q)(echo "#!$(TCLSH)")>|$@
+	$(Q)(echo '#!/usr/bin/env tclsh')>|$@
 	$(Q)(echo "source $(EMC2_TCL_LIB_DIR)/ngcgui.tcl")>>$@
 	$(Q)(echo "::ngcgui::standalone_ngcgui")>>$@
 	$(Q)chmod +x $@
@@ -500,7 +500,7 @@ TARGETS += ../bin/ngcgui
 TARGETS += ../bin/pyngcgui
 ../bin/pyngcgui: Makefile
 	$(Q)rm -f $@
-	$(Q)(echo "#!$(PYTHON)")>|$@
+	$(Q)(echo '#!/usr/bin/env python3')>|$@
 	$(Q)(echo "import pyngcgui")>>$@
 	$(Q)(echo "pyngcgui.standalone_pyngcgui()")>>$@
 	$(Q)chmod +x $@
@@ -509,7 +509,7 @@ TARGETS += ../bin/pyngcgui
 TARGETS += ../bin/gremlin_view
 ../bin/gremlin_view: Makefile
 	$(Q)rm -f $@
-	$(Q)(echo "#!$(PYTHON)")>|$@
+	$(Q)(echo '#!/usr/bin/env python3')>|$@
 	$(Q)(echo "import gremlin_view")>>$@
 	$(Q)(echo "gremlin_view.standalone_gremlin_view()")>>$@
 	$(Q)chmod +x $@
@@ -518,7 +518,7 @@ TARGETS += ../bin/gremlin_view
 TARGETS += ../bin/halshow
 ../bin/halshow: Makefile
 	$(Q)rm -f $@
-	$(Q)(echo "#!$(TCLSH)")>|$@
+	$(Q)(echo '#!/usr/bin/env tclsh')>|$@
 	$(Q)(echo "source $(EMC2_TCL_LIB_DIR)/bin/halshow.tcl")>>$@
 	$(Q)chmod +x $@
 
